src/common/components/cards/rule-resources.tsx

```diff
@@ -8,6 +8,11 @@ import { UnifiedRule } from 'common/types/store-data/unified-data-interface';
 import { isEmpty } from 'lodash';
 import * as React from 'react';
 
+import { InstanceOutcomeType } from '../../../reports/components/instance-outcome-type';
+import {
+    getNeedsReviewRuleResourcesUrl,
+    isOutcomeNeedsReview,
```

Comment:
For testing purposes, should `isOutcomeNeedsReview` be passed in as part of `RuleResourcesDeps`? That would decouple the consumption of `isOutcomeNeedsReview` from the implementation and allow the 2 ends to be tested independently.

Reply:
resolved the comment and passed  isOutcomeNeedsReview and getNeedsReviewRuleResourcesUrl as parts of RuleResourcesDeps